Makefile

```diff
@@ -33,6 +36,22 @@ clean-wheels:
 wheels:
 	./build-wheels
 
+ansible-update: clean-wheels wheels ansible-wheel-update ansible-requirements
+
+ansible-wheel-update:
+	rm -fr $(ANSIBLE_DESTDIR)/wheels
+	mkdir -p $(ANSIBLE_DESTDIR)/wheels
+	cp -vf wheels/*.whl $(ANSIBLE_DESTDIR)/wheels/
+
+ansible-requirements: requirements.txt requirements-siptools.txt
```

Comment:
requirements-siptools.txt was deleted

Reply:
Thanks for noticing! Have to fix this...

Reply:
I had it removed already, but resolved conflict incorrectly in a rebase.

It's now fixed.